requirements.txt

```diff
@@ -9,15 +9,16 @@ tox
 # s2_inference:
 more_itertools
```

Comment:
Can you revert changes in this file? Changing requirements makes it liable to break builds